src/middlewared/middlewared/alembic/versions/24.04/2023-10-30_21-02_readonly_privilege.py

```diff
@@ -0,0 +1,31 @@
+"""empty message
+
+Revision ID: 649fe0f7f0ba
+Revises: e3436f15aa9c
+Create Date: 2023-10-30 21:02:00.757892+00:00
+
+"""
+from alembic import op
+import sqlalchemy as sa
+
+
+# revision identifiers, used by Alembic.
+revision = '649fe0f7f0ba'
+down_revision = 'e3436f15aa9c'
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    # ### commands auto generated by Alembic - please adjust! ###
+    op.execute(
+        "INSERT INTO account_privilege (builtin_name, name, local_groups, ds_groups, allowlist, web_shell, roles) "
+        "VALUES ('READONLY', 'Readonly Users', '[]', '[]', '[]', 0, '[\"READONLY\"]')"
```

Comment:
I believe per our previous conversations about sensitive info being exposed to READONLY users that we may want to explicitly name something like "READONLY ADMINISTRATORS", correct?